packages/frontend/src/components/wallet-migration/modals/RotateKeysModal/RotateKeysModal.jsx

```diff
@@ -0,0 +1,300 @@
+import { KeyPair } from 'near-api-js';
+import { generateSeedPhrase } from 'near-seed-phrase';
+import React, {useState, useEffect, useMemo, useRef} from 'react';
+import { Translate } from 'react-localize-redux';
+import { useDispatch, useSelector } from 'react-redux';
+import styled from 'styled-components';
+import { useImmerReducer } from 'use-immer';
+
+import { NETWORK_ID } from '../../../../config';
+import { switchAccount } from '../../../../redux/actions/account';
+import { showCustomAlert } from '../../../../redux/actions/status';
+import { selectAccountId } from '../../../../redux/slices/account';
+import WalletClass, { wallet } from '../../../../utils/wallet';
+import AccountListImport from '../../../accounts/AccountListImport';
+import { IMPORT_STATUS } from '../../../accounts/batch_import_accounts';
+import sequentialAccountImportReducer, { ACTIONS } from '../../../accounts/batch_import_accounts/sequentialAccountImportReducer';
+import ConfirmPassphrase from '../../../accounts/recovery_setup/new_account/ConfirmPassphrase';
+import SavePassphrase from '../../../accounts/recovery_setup/new_account/SavePassphrase';
+import FormButton from '../../../common/FormButton';
+import LoadingDots from '../../../common/loader/LoadingDots';
+import Modal from '../../../common/modal/Modal';
+import { WALLET_MIGRATION_VIEWS } from '../../WalletMigration';
+
+const ButtonsContainer = styled.div`
+    text-align: center;
+    width: 100% !important;
+    display: flex;
+`;
+
+const StyledButton = styled(FormButton)`
+    width: calc((100% - 16px) / 2);
```

Comment:
Can we avoid setting a width on buttons like this and instead utilize flexbox or similar for button layout?

Reply:
Yes, I believe all modals use this Styled Button and Container styled element, so would be great to have a followup PR to clean all of this up in one go.